ReSharper.FSharp/test/data/parsing/Type member - Abstract 04 - Constrained type usage.fs

```diff
@@ -0,0 +1,4 @@
+module Module
+
+type T =
+    abstract member Using: resource:'T * binder:('T -> Async<'U>) -> Async<'U> when 'T :> System.IDisposable
```

Comment:
@DedSec256 Could you please simplify the test, so it's easier to see what exactly being tested here?